<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoResultadoConsulta</class>
 <widget class="QWidget" name="DialogoResultadoConsulta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogoResultadoConsulta</string>
  </property>
  <widget class="QTabWidget" name="analisis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>421</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="pestania_1">
    <attribute name="title">
     <string>Fuerza en noticia</string>
    </attribute>
    <widget class="QWidget" name="fuerza_en_noticia_vacio" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>391</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="lbl_sin_valores_1">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>291</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>sin valores</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="pestania_2">
    <attribute name="title">
     <string>Sentimiento</string>
    </attribute>
    <widget class="QWidget" name="sentimiento_vacio" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>391</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="lbl_sin_valores_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>291</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>sin valores</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QSlider" name="anios">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>611</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
  <widget class="QDateEdit" name="calendario">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>435</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::YearSection</enum>
   </property>
  </widget>
  <widget class="QSlider" name="meses">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>485</y>
     <width>611</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSlider" name="dias">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>611</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
